### DISCUSSION TITLE
Shell wrapper: zoxide no match found

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [sxyazi/yazi#1287] [1. Q&A]
![image](https://github.com/user-attachments/assets/d7f4551e-0170-4e63-8af7-b2689797e919)

**Comment by xfzv** [comment]
Sharing more details about your setup would help, e.g. `yazi -V` output, what shell you're using, your actual `yy` function, how you load `zoxide`...

Not sure why `yy` is producing some output, it's completely silent here.

Is `zoxide: no match found` still displayed if you temporarily disable `zoxide`? I don't think it's required for `yy` to work, at least it's not mentioned in the [docs](https://yazi-rs.github.io/docs/quick-start/#shell-wrapper).

**Reply by sachinsenal0x64** [reply]
> Sharing more details about your setup would help, e.g. `yazi -V` output, what shell you're using, your actual `yy()` function, how you load `zoxide`...
> 
> Not sure why `yy` is producing some output, it's completely silent here.
> 
> Is `zoxide: no match found` still displayed if you temporarily disable `zoxide`? I don't think it's required for `yy()` to work, at least it's not mentioned in the [docs](https://yazi-rs.github.io/docs/quick-start/#shell-wrapper).


I use the Fish shell with the latest version of Yazi. And I'm currently using zoxide instead of CD


`Not sure why yy is producing some output, it's completely silent here.
`

The reason is that I added some print statements for debugging purposes.

**Comment by sachinsenal0x64** [comment]
So when I removed the cat alias, it worked as it did before when I had the alias with bat, lol.
